libcontainerd/container_windows.go

```diff
@@ -42,13 +42,18 @@ func (ctr *container) start() error {
 	// until the container is done with the servicing execution.
 	logrus.Debugln("Starting container ", ctr.containerID)
 	if err = ctr.hcsContainer.Start(); err != nil {
-		logrus.Errorf("Failed to start compute system: %s", err)
+		logrus.Errorf("Failed to start container: %s", err)
+		if err := ctr.terminate(); err != nil {
+			logrus.Errorf("Failed to cleanup after a failed Start. Ignoring this. %s", err)
```

Comment:
It is being ignored because the Start error above is the more important of the two errors to return, but I believe it is an Error nonetheless. Should I remove the "Ignoring this" text in the error, or does it make more sense to just leave it as a Debug?

I'm fine either way, but I do agree with @LK4D4 that this is more of an Error than something for the Debug logs.